src/EventListeners.ts

```diff
@@ -654,7 +654,7 @@ TODO: Fix this entire tab it's utter shit
     DOMCacheGetOrSet('singularityPerksGrid').append(perkHTML)
     DOMCacheGetOrSet(perk.ID).addEventListener('mouseover', () => {
       const perkInfo = getLastUpgradeInfo(perk, player.highestSingularityCount)
-      const levelInfo = `${i18next.t('general.level')} ${perkInfo.level} - (Singularity ${perkInfo.singularity})`
+      const levelInfo = `${i18next.t('general.level')} ${perkInfo.level} - (${i18next.t('general.singularity')} ${perkInfo.singularity})`
```

Comment:
this needs to be a singular translation too

Reply:
Ditto to above comment